src/Riter/Core/GlobalHotkeyManager.cs

```diff
@@ -44,15 +44,15 @@ public GlobalHotkeyManager(Window window)
     /// <param name="callback">The method to invoke when the hotkey is pressed.</param>
     /// <returns><c>true</c> if the hotkey is successfully registered; otherwise, <c>false</c>.</returns>
 
-    public bool RegisterHotkey(int id, uint modifiers, uint key, Action callback)
+    public bool RegisterHotkey(HotKey id, uint modifiers, uint key, Action<HotKey> callback)
```

Comment:
consider renaming the callback parameter to something like hotkeyAction for better clarity.